src/App.js

```diff
@@ -4,20 +4,31 @@ import './App.scss';
 import todos from './api/todos';
 import users from './api/users';
 
+import { TodoList } from './components/TodoList';
+
+const prepareTodos = todos.map(todo => ({
```

Comment:
prepareTodos - название функции: подготовьЗадачи
preparedTodos - название данных: подготовленныеЗадачи

```suggestion
const preparedTodos = todos.map(todo => ({
```